fixed-price-sale/program/src/processor/create_market.rs

```diff
@@ -65,6 +66,16 @@ impl<'info> CreateMarket<'info> {
                     }
                 }
             }
+
+            if remaining_accounts.len() != 1 {
+                return Err(ErrorCode::CollectionMintMissing.into());
+            }
+
+            let collection_mint = &remaining_accounts[0];
+
+            if collection_mint.key != &gating_data.collection || collection_mint.owner != &spl_token::id() {
```

Comment:
gating_data.collection is required field.
And not sure that we can check if collection if verified because it's collection mint itself. I mean that token doesn't belong to any collection.
Correct me please if I missed smth